src/main/resources/intentionDescriptions/AddWildcardArmIntention/description.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <body>
-Adds `_` pattern in <code>match</code> expression if the expression is not exhaustive,
+Adds <code>`_`</code> pattern in <code>match</code> expression if the expression is not exhaustive,
```

Comment:
```suggestion
Adds <code>_</code> pattern in <code>match</code> expression if the expression is not exhaustive,
```

From my point of view, `<code>` is enough

Reply:
I actually have no idea which way these snippets should _really_ be formatted. Does it support markdown? Is the html required? E.g. what exactly would that backtick inside of `<code/>` do? There are other cases of backticks within code tags in the descriptions, should I fix it or it doesn't matter?

Also, some of the descriptions are just a single line of simple text, while some use the heavyweight (RsDetachedFile.html)
```html
<html>
<body>
Detects files that are not attached to any Rust module.
</body>
</html>
```

It looks like that `<body/>` tag isn't doing anything... or is it? Should we normalize the descriptions to some form?

Reply:
> Does it support markdown?

No

> Is the html required? E.g. what exactly would that backtick inside of <code/> do?

Yep. Descriptions are considered as html even if `html` tag is not written. So  `` <code>`</code> `` renders `` ` `` with monospace font

> There are other cases of backticks within code tags in the descriptions, should I fix it or it doesn't matter?

Actually, it would be nice to fix all such things. Probably, we can even write some checks in tests to suggest using `<code>` tag instead of `` ` `` (just an idea from the top of my head. not sure it's a good idea)

> Also, some of the descriptions are just a single line of simple text, while some use the heavyweight
> It looks like that <body/> tag isn't doing anything... or is it? Should we normalize the descriptions to some form?

I haven't checked all details, but it seems that you always can omit `html` and `body` tags and IDE will wrap the text itself (see [here](https://github.com/JetBrains/intellij-community/blob/a74e0509a5da093d1fb8eb58c73aa0943260ea80/platform/lang-impl/src/com/intellij/profile/codeInspection/ui/DescriptionEditorPane.kt#L38)). But I'd still prefer to wrap descriptions into `html` and `body` tags just because the platform uses this style and probably there are some corner cases related to it